src/libs/antares/study/parts/thermal/container.cpp

```diff
@@ -99,10 +96,9 @@ uint PartThermal::prepareClustersInMustRunMode()
     do
```

Comment:
This loop do ... while is complicated. Why is it there ? Is it necessary or not ?
We'll to think about it some day.